db/models.py

```diff
@@ -105,7 +105,7 @@ class EVCUpdateDoc(DocumentBaseModel):
     service_level: Optional[int] = None
     circuit_scheduler: Optional[list[CircuitScheduleDoc]] = None
     metadata: Optional[dict] = None
-    enabled: Optional[bool] = None
+    enable: Optional[bool] = None
```

Comment:
This will impact on how EVCs are loaded from the DB to in memory, we'd need scripts, but I think we don't need to make this change here, check out the comment below, or let me know if you had other reasons to try to go for this approach.

Reply:
I had a blocker in [hasattr](https://github.com/kytos-ng/mef_eline/blob/2e45177240ec9e9a94204494f99f90d7e3e94cf7/models/evc.py#L239-L240). Turned out it returned true from `asattr(self, "enable")` because of the method from `GenericEntity`: `enable()`. Removed it. Now using a set `updatable_attributes` to discard any invalid field.